etc/foreman.yaml.example

```diff
@@ -2,9 +2,13 @@
 :proto: https
 :host: foreman.example.com
 :port: 443
+# if using http with ssl certificates
 :ssl_ca: "/var/lib/puppet/ssl/certs/ca.pem"
 :ssl_cert: "/var/lib/puppet/ssl/private_keys/foreman.example.com.pem"
 :ssl_key: "/var/lib/puppet/ssl/certs/foreman.example.com.pem"
+# if using http with username and password instead of https with certicates
+:username: admin
+:password: changeme
```

Comment:
You can still set the URL to https, but you wouldn't need to be providing a key/cert at the same time